### DISCUSSION TITLE
Color Vite

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#3270] [Help] [answered]
How can I update the color of my site using Vite, I followed the instructions and it does not change from yellow. thanks

https://filamentphp.com/docs/2.x/admin/appearance#building-themes

**Comment by danharrin** [comment]
Please share your Tailwind config

**Reply by jcgmalave** [reply]
I followed all the steps in the guide, and then I did the npm run dev but I'm not sure where I should specify the new color

**Comment by jcgmalave** [comment, marked as answer]
In order to make it work with VITE i had to do the next steps:

npm install -g npm@8.16.0
npm update
follow the instructions from: 
		https://filamentphp.com/docs/2.x/admin/appearance#building-themes
npm run build  
npm run dev

Hope it can help u guys
